### ISSUE TITLE
EnderLink Chest can't upgrade

### ISSUE THREAD

**Issue by litjohn50** [ShetiPhian/Localizations-and-Issues#145]
I am trying to upgrade the Chest by rightclicking with diamond .
https://gyazo.com/8cb7feb5452991e38d4b0720c5e040c9

or even enderpearls.
https://gyazo.com/bd89ceaf7eb43e8c4357106840b50ad6

Forge 14.23.2.2613
Verision Multistorage 1.12.0-1.4.7

**Comment by ShetiPhian** [comment]
I just tested in single player with the same versions and it works.

Might be due to another mod canceling the event.

This build will receive the event even if its canceled:
http://www.mediafire.com/file/dwo2a73zexn7vak/multistorage-1.12.0-1.4.8-TESTER.jar

**Comment by litjohn50** [comment]
https://gyazo.com/7b57218d35a6df7a95d7090cab3aa42a 

https://gyazo.com/7b57218d35a6df7a95d7090cab3aa42a

Tried it with your  tester version didn't work
[latest.log](https://github.com/ShetiPhian/Localizations-and-Issues/files/1734966/latest.log)

**Comment by ShetiPhian** [comment]
I re-watched your videos, and noticed something I missed before, the GUI shouldn't open.
Its coded not to open if the player is sneaking. Even with an empty hand you can not open the GUI if your sneaking.

In the first one with the diamond it looks like you stop sneaking right before right clicking. Same thing in your second video. Your latest video happens too fast but likely the same problem because the GUI opens.

Are you holding down the sneak key while clicking? if so it appears like something is canceling sneak on you when you click. likely you won't be able to sneak and place blocks either.

With an empty hand, hold sneak and click the chest. Nothing should happen.
If that works as intended, try with a diamond in hand.
If that doesn't make the chest private, but also doesn't open the GUI, you'll need to look at configs (Settings>EnderLink)

**Comment by litjohn50** [comment]
It looks that I stop sneeking but your hand doesn't move when shifting a block with a diamond. Each video i right clicked 6 times.

**Comment by ShetiPhian** [comment]
I tried typing out a troubleshooting walk through, but it had too many possible side tangents and I just couldn't get it to display in a clean easy to follow manner.

Question:
Is this a public modpack? or one with other players?
-- Are other players having the same issue.

Request:
Please provide the multistorage.cfg file, its possible the problem is there.

Quick Test:
Create a new Minecraft instance with only Forge, ShetiPhianCore and MultiStorage
Use new downloads and do not copy configs, let them generate the defaults
Can you upgrade the chests there?

**Comment by litjohn50** [comment]
https://minecraft.curseforge.com/projects/towncraft public modpack play on a server. Noone can update their ender chests.
 
Config https://pastebin.com/DdV4hyBA

**Comment by ShetiPhian** [comment]
Found it,
Something in that pack is contaminating the ore dictionary items by adding nbt data.

My matcher is nbt sensitive, so this causes it to think you have the wrong item. I've rewrote my code to no longer be nbt sensitive. (limits control over what items are used, but I don't think that will be an issue)

These builds worked for me.
http://www.mediafire.com/file/vimlmdi9abvada1/endertanks-1.12.0-1.6.4.jar
http://www.mediafire.com/file/cyc95dn4y3eey7q/multistorage-1.12.0-1.4.8.jar

If they work on your end, in single player, I'll push them to Curse.
Then the pack maker will need to update to them, then your server and other players will need to update to the new pack version.
